client/packages/common/src/ui/forms/JsonForms/api/api.ts

```diff
@@ -2,6 +2,7 @@ import { DocumentRegistryNodeContext } from '@common/types';
 import {
   DocumentFragment,
   DocumentRegistryFragment,
+  EncounterExtractFieldFragment,
```

Comment:
and for the method - I don't think `extract` adds any information to the name. `encounterFields` and `encounterFieldFragment` are fine by me

Reply:
Good idea renamed it in backend + frontend